store/postgres.go

```diff
@@ -49,40 +55,48 @@ type postgres struct {
 	pool *pgxpool.Pool
 }
 
-func (p *postgres) Apply(schema *bubblySchema) error {
+func (p *postgres) Apply(tenant string, schema *bubblySchema) error {
```

Comment:
Shouldn't the _schema_ know what `tenant` it is for? As in, our current `bubblySchema` data type just assumes that there is a single tenant and now we are implementing the multitenancy by providing a tenant name everywhere as a string, but maybe it could be a part of `bubblySchema`? There is no `bubblySchema` without a tenant, right? Even if that tenant is the "default" one. 

Just a thought.

Reply:
Thanks for the excellent thought! Well it's a design decision whether we want the `bubblySchema` to know who it belongs to, or simply to store the schema and whoever uses it does so to the correct tenant. I think there are more changes coming to this, in fact the `bubblySchema` is somewhat redundant as we have the `schemaGraph` so I vote to keep it like this for now. We can see how things materialise later.